decidim-meetings/spec/system/meeting_registrations_spec.rb

```diff
@@ -313,13 +313,9 @@ def questionnaire_public_path
         it "shows errors for invalid file" do
           visit questionnaire_public_path
 
-          dynamically_attach_file("questionnaire_responses_0_add_documents", Decidim::Dev.asset("verify_user_groups.csv"))
+          dynamically_attach_file("questionnaire_responses_0_add_documents", Decidim::Dev.asset("verify_user_groups.csv"), keep_modal_open: true)
 
-          expect(page).to have_field("public_participation", checked: false)
-          find_by_id("questionnaire_tos_agreement").set(true)
-          accept_confirm { click_on "Submit" }
-
-          expect(page).to have_content("Needs to be reattached")
+          expect(page).to have_content("Validation error!")
```

Comment:
To be done in `./decidim-core/config/locales/en.yml`, can you improve this error 🙏🏽? As we're changing the behavior and we leave this too generic error, I think it could be difficult to understand what's happening. How about "Validation error! Check that the file has an allowed extension"

Reply:
That sounds right